lib/luaxtable.c

```diff
@@ -88,6 +192,15 @@ static const lunatik_class_t luaxtable_class = {
 	.sleep = false,
 };
 
+static inline void luaxtable_setbuffer(lua_State *L, int idx, luaxtable_t *xtable)
```

Comment:
it's actually `newbuffer`, right? as you are creating new one bellow on every call..

Reply:
we are reusing the skb luadata object, hence I name it setbuffer.

Reply:
Ah, yes, got your point, it should be a `newbuffer`, but since I was using `set{integer,string}` for setting xtable values, I used it.

Reply:
I meant, we are always creating a new luadata object (therefore, a buffer).. I think `set` is misleading here.. `new` would be clearer..

Reply:
Yes, I will change it in the next push